command.json

```diff
@@ -35,6 +35,13 @@
           "description": "Company address",
           "required": false,
           "prompt": true
+        },
+        {
+          "name": "adminPassword",
```

Comment:
question: is this/should this be camel or kebab case? I kind of feel that camel case is a bit odd in a CLI which I think is what this command name would become no?

Reply:
I was also on the fence with this one. We currently don't have any camel or kebab case names in  `command.json`. I went with camel, because it's a json file and will not be shown to the user on the CLI.

I'm fine with kebab too.

Reply:
isn't it the actual input though? like wouldn't it be `bedrock cloud adminPassword`? If that's the case, doesn't `bedrock cloud admin-password` seem more appropriate?

I also take heavy and violent issue with the statement "I went with camel because it's a JSON file" :)

Reply:
No, this is to prompt you during the `cloud create` process to pass an admin password. So the use only sees:

`? Enter admin password (optional): ›`

I'll take back my JSON file statement.